### PR TITLE
doc: Update Configurator steps

### PR DESCRIPTION
**Pull request by amyreye**
- Add build steps
- Update steps per new UI

Signed-off-by: Reyes, Amy <amy.reyes@intel.com>

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

File: doc/getting-started/getting-started.rst
Comment (gvancuts) [inline doc/getting-started/getting-started.rst:456]: Any particular reason we're not using `pip3` directly here? I.e. `pip install -r requirements.txt`

File: doc/getting-started/getting-started.rst
Comment (gvancuts) [inline doc/getting-started/getting-started.rst:527]:
```suggestion
      configuration file and launch scripts.
```
? Or is there just one launch script to be generated?

File: doc/getting-started/getting-started.rst
Comment (gvancuts) [inline doc/getting-started/getting-started.rst:506]:
```suggestion
#. Generate the scenario configuration file and launch scripts:
```
? Or is there just one launch script to be generated?

File: doc/getting-started/getting-started.rst
Comment (dbkinder) [inline doc/getting-started/getting-started.rst:184]: Eventually this will checkout the v3.0 branch

File: doc/getting-started/getting-started.rst
Comment (dbkinder) [inline doc/getting-started/getting-started.rst:187]: eventually the acrn-kernel branch name would be release_3.0

File: doc/getting-started/getting-started.rst
Comment (dbkinder) [inline doc/getting-started/getting-started.rst:416]: What new terminal?
```suggestion
   #. Rerun your ``.bashrc`` initialization script and then install the latest version of Node.js and npm:
```

File: doc/getting-started/getting-started.rst
Comment (dbkinder) [inline doc/getting-started/getting-started.rst:436]:
```suggestion
   When prompted by the Rust installation script, type ``1`` and press :kbd:`Enter`.
```

File: doc/getting-started/getting-started.rst
Comment (dbkinder) [inline doc/getting-started/getting-started.rst:451]:
```suggestion
#. Install additional ACRN Configurator dependencies:
```

File: doc/getting-started/getting-started.rst
Comment (dbkinder) [reply doc/getting-started/getting-started.rst:506]: Just one script (one post-launched VM) in this new GSG

**Review by dbkinder — CHANGES_REQUESTED** [review]

File: doc/getting-started/getting-started.rst
Comment (amyreye) [reply doc/getting-started/getting-started.rst:456]: @gvancuts Not sure, this command came from Eng team. Should I change it to ``sudo pip3 install -r requirements.txt``?

Comment (gvancuts) [reply doc/getting-started/getting-started.rst:460]: It looks like this is in fact the most robust mechanism to use - see also https://github.com/projectacrn/acrn-hypervisor/pull/7358#discussion_r858672890

Let's keep it as is (and perhaps we should update all `pip install ...` commands to this form (`python3 -m pip install...`) throughout our documentation.

**Comment by amyreye** [comment]
@NanlinXie Can you let me know when the Eng team has validated this PR, and we can remove the "On Hold" label?

File: doc/getting-started/getting-started.rst
Comment (NanlinXie) [reply doc/getting-started/getting-started.rst:184]: We could use latest tag "acrn-2022w18.4-180000p" for now.

File: doc/getting-started/getting-started.rst
Comment (NanlinXie) [reply doc/getting-started/getting-started.rst:187]: We could use latest stable tag "acrn-2022w18.4-180000p" for now.

**Review by NanlinXie — APPROVED** [review]

**Comment by NanlinXie** [comment]
@amyreye , I removed the "On Hold" label, and I think we could continue on merging and public document update. Our validation could based on public document and we could file new PR to update once any doc issue found.

**Comment by amyreye** [comment]
> @amyreye , I removed the "On Hold" label, and I think we could continue on merging and publishing document update. Our validation could based on public document and we could file new PR to update once any doc issue found.

@NanlinXie Thank you. One concern I have is that I get an error when trying to build ACRN with the board XML and scenario XML generated from the Configurator with both master branch and tag "acrn-2022w18.4-180000p". I will file a ticket. Can we merge this PR, even if the build may be broken? What do you think, @dbkinder?

File: doc/getting-started/getting-started.rst
Comment (amyreye) [reply doc/getting-started/getting-started.rst]: Done.

File: doc/getting-started/getting-started.rst
Comment (amyreye) [reply doc/getting-started/getting-started.rst]: Done

File: doc/getting-started/getting-started.rst
Comment (dbkinder) [inline doc/getting-started/getting-started.rst:570]: Maybe we should change these references now to be ``acrn-my_board-shared-*.deb`` ?  Does that match the file names you're seeing for the hypervisor build output?

**Review by dbkinder — CHANGES_REQUESTED** [review]

File: doc/getting-started/getting-started.rst
Comment (amyreye) [reply doc/getting-started/getting-started.rst:570]: Unfortunately, I haven't been able to build successfully, so I'm not sure what the file name is. I filed a ticket in JIRA with the error details.

Comment (dbkinder) [reply doc/getting-started/getting-started.rst:570]: In that case, let's just go with it as it is and fix later.

**Review by dbkinder — APPROVED** [review]
We'll need to fix some filenames and branch names ...

**Pull request merged**
